audio-analysis/main.py

```diff
@@ -1,10 +1,40 @@
 import videoHelper
+import pandas as pd
 
-trumpVideo = videoHelper.videoObject('https://www.youtube.com/watch?v=JZRXESV3R74', False)
-print(trumpVideo.getFilename())
-trumpVideo.getAudio()
-trumpVideo.getTextAnalysis()
-trumpVideo.getAmplitudeAnalysis()
+# Save Video function creates a video object and outputs all the nice analysis stuff
+# Will add a function to consolidate video data into a dictionary 
+# Will add a function to push video dictionaries to SQL table
+def saveVideo(ytURL, hTag):
+    videoX = videoHelper.videoObject(ytURL, hTag)
+    print(videoX.getFilename())
```

Comment:
just a nit but can we change some variables names ... right now looks like a sPonGeBob meme:
ytURl -> url
hTag -> tag
videoX -> video